src/manifolds/Symplectic.jl

```diff
@@ -0,0 +1,930 @@
+@doc raw"""
+    Symplectic{n, ℝ} <: AbstractEmbeddedManifold{ℝ, DefaultIsometricEmbeddingType}
+
+Over the real number field ℝ the elements of the symplectic manifold
```

Comment:
For completeness we could also document the tangent space here.

Reply:
:+1: